app/services/invite_employee_service.rb

```diff
@@ -0,0 +1,63 @@
+class InviteEmployeeService < ApplicationService
```

Comment:
Changed check_employee_association service to invite_employee_service, since the association is done only after the invite is accepted